### DISCUSSION TITLE
What is the plan?

### DISCUSSION THREAD

**Discussion by Wardormeur** [parcelvoy/platform#471] [General]
Hello :)
We're wondering whether to adopt parcelvoy for our company instead of some competitor like Novu.
Would you be inclined to share where the project is willing to go, like a roadmap or your personal thoughts?
Thanks a lot

**Comment by pushchris** [comment]
@Wardormeur its all based on what your needs are. Novu is a very different product aimed more at developers. Parcelvoy is aimed at marketing departments and product teams so that they can interact with users without having to suck developers time.

Upcoming developments are in-app messaging and modals as well as continued performance improvements.

**Comment by Wardormeur** [comment]
Thanks a lot for your answer and the upcoming changes!
It definitely helped in making a choice
